### PR TITLE
KAA-1677: Set Kaa version to 0.10.1 for release-0.10 branch

### PR DESCRIPTION
**Pull request by vtkhir**

### CODE REVIEW

File: server/appenders/oracle-nosql-appender/src/main/java/org/kaaproject/kaa/server/appenders/oraclenosql/config/gen/KvStoreNode.java [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
+package org.kaaproject.kaa.server.appenders.oraclenosql.config.gen;  
 @SuppressWarnings("all")
 @org.apache.avro.specific.AvroGenerated
 public class KvStoreNode extends org.apache.avro.specific.SpecificRecordBase implements org.apache.avro.specific.SpecificRecord {
   public static final org.apache.avro.Schema SCHEMA$ = new org.apache.avro.Schema.Parser().parse("{\"type\":\"record\",\"name\":\"KvStoreNode\",\"namespace\":\"org.kaaproject.kaa.server.appenders.oraclenosql.config.gen\",\"fields\":[{\"name\":\"host\",\"type\":{\"type\":\"string\",\"avro.java.string\":\"String\"},\"displayName\":\"Host\",\"weight\":0.75,\"by_default\":\"localhost\"},{\"name\":\"port\",\"type\":\"int\",\"displayName\":\"Port\",\"weight\":0.25,\"by_default\":5000}]}");
-  private java.lang.String host;
-  private int port;
+  public static org.apache.avro.Schema getClassSchema() { return SCHEMA$; }
+   private java.lang.String host;
+   private int port;
```
Comment (rasendubi) [inline server/appenders/oracle-nosql-appender/src/main/java/org/kaaproject/kaa/server/appenders/oraclenosql/config/gen/KvStoreNode.java:13]: Has this been auto-generated? Indentation looks wrong.

File: server/verifiers/gplus-verifier/src/main/java/org/kaaproject/kaa/server/verifiers/gplus/config/gen/GplusAvroConfig.java [hunk 2]
```diff
@@ -1,25 +1,24 @@
...
 @org.apache.avro.specific.AvroGenerated
 public class GplusAvroConfig extends org.apache.avro.specific.SpecificRecordBase implements org.apache.avro.specific.SpecificRecord {
   public static final org.apache.avro.Schema SCHEMA$ = new org.apache.avro.Schema.Parser().parse("{\"type\":\"record\",\"name\":\"GplusAvroConfig\",\"namespace\":\"org.kaaproject.kaa.server.verifiers.gplus.config.gen\",\"fields\":[{\"name\":\"max_parallel_connections\",\"type\":\"int\",\"displayName\":\"Max connections\",\"by_default\":\"20\"},{\"name\":\"min_parallel_connections\",\"type\":\"int\",\"displayName\":\"Min connections\",\"by_default\":\"2\"},{\"name\":\"keep_alive_time_milliseconds\",\"type\":\"long\",\"displayName\":\"Connection lifetime (ms)\",\"by_default\":\"60000\"}]}");
-  private int max_parallel_connections;
-  private int min_parallel_connections;
-  private long keep_alive_time_milliseconds;
+  public static org.apache.avro.Schema getClassSchema() { return SCHEMA$; }
+   private int max_parallel_connections;
+   private int min_parallel_connections;
+   private long keep_alive_time_milliseconds;
```
Comment (rasendubi) [inline server/verifiers/gplus-verifier/src/main/java/org/kaaproject/kaa/server/verifiers/gplus/config/gen/GplusAvroConfig.java:14]: Same here.

**Review by rasendubi — APPROVED** [review]
Quick question. Otherwise, looks good.

File: server/appenders/oracle-nosql-appender/src/main/java/org/kaaproject/kaa/server/appenders/oraclenosql/config/gen/KvStoreNode.java (hunk 1 above)
Comment (vtkhir) [reply server/appenders/oracle-nosql-appender/src/main/java/org/kaaproject/kaa/server/appenders/oraclenosql/config/gen/KvStoreNode.java:13]: yes it was auto-generated

File: server/verifiers/gplus-verifier/src/main/java/org/kaaproject/kaa/server/verifiers/gplus/config/gen/GplusAvroConfig.java (hunk 2 above)
Comment (vtkhir) [reply server/verifiers/gplus-verifier/src/main/java/org/kaaproject/kaa/server/verifiers/gplus/config/gen/GplusAvroConfig.java:14]: auto-generated

**Pull request merged**